tests/test_appliancesmanager.py

```diff
@@ -0,0 +1,148 @@
+import asyncio
+
+import pytest
+
+from whirlpool.appliancesmanager import AppliancesManager
+from whirlpool.auth import Auth
+
+from .aiohttp import AiohttpClientMocker
+from .mock_backendselector import BackendSelectorMock
+from .utils import (
+    get_mock_coro,
+    mock_appliancesmanager_get_account_id_get,
+    mock_appliancesmanager_get_owned_appliances_get,
+    mock_appliancesmanager_get_shared_appliances_get,
+)
+
+BACKEND_SELECTOR_MOCK = BackendSelectorMock()
+
+
+async def test_create_headers_includes_wp_client_brand(
+    http_client_mock: AiohttpClientMocker,
+):
+    http_client_mock.create_session(asyncio.get_event_loop())
+    auth = Auth(BACKEND_SELECTOR_MOCK, "email", "secretpass", http_client_mock.session)
+    auth._auth_dict["access_token"] = "acess_token_123"
+
+    am = AppliancesManager(BACKEND_SELECTOR_MOCK, auth, http_client_mock.session)
+
+    headers = am._create_headers()
+
+    assert headers["WP-CLIENT-BRAND"] == BACKEND_SELECTOR_MOCK.brand.name
+
+    await http_client_mock.close_session()
+
+
+async def test_account_id_set_if_populated_in_auth_dict(
```

Comment:
As in the above test, I think we can test this using only the public fetch_appliances().
We could have the "normal" `test_fetch_appliances()` that uses the one from the auth_dict, since that is the expected use case, and another `test_fetch_appliances_no_account_id_on_auth()` or something, that tests the other case